deploy-staging.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+staging_dir="/var/www/staging"
+repo_owner=lowellmakes
+repo_name=3dprinterstatus
+workflow_name=php.yml
+branch_name=main
+repo_url="https://github.com/${repo_owner}/${repo_name}.git"
+github_api_url="https://api.github.com/repos/${repo_owner}/${repo_name}/actions/workflows/${workflow_name}/runs"
+
+while true; do
```

Comment:
❓ Is the idea that you'll start this manually once, then it constantly polls this repo for new (successful) commits and pulls them down to the working copy?